css/main.css

```diff
@@ -325,3 +326,67 @@ li {
 .skills {
   list-style: inside url("../media/svg/circle.svg");
 }
+
+/* My Footer */
+
+.contact-container {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 120px 24px;
+}
+
+.contact-title {
+  font-family: Inter, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  color: #172b4d;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0;
+  margin: 0 auto;
+}
+
+form input {
+  border-bottom: 1px solid #dbd8d7;
+  color: #979493;
+}
+
+form input,
+form textarea {
+  font-size: 15px;
+  line-height: 24px;
+  margin: 24px 0;
+  padding: 10px 2px;
+}
+
+form textarea {
+  background-color: #fbf8f7;
+}
+
+form input:focus,
+form textarea:focus {
+  outline: none;
+  color: black;
+}
+
+.btn {
+  display: flex;
```

Comment:
- [ ] [required] CSS codes should be DRY (Don't Repeat Yourself), there are some code repetitions here, `display: flex;` => line 333, 350, and 381, `text-align: center;` is repeated as well. I suggest you fix this by creating a utility class which holds the properties you are repeating and then apply that class in html wherever you need those properties. This will help you in case you need to maintaining or debug your code

Reply:
Done